plugins/network-elements/nuage-vsp/src/com/cloud/network/element/NuageVspElement.java

```diff
@@ -367,22 +483,16 @@ public boolean applyStaticNats(Network config, List<? extends StaticNat> rules)
             sourceNatDetail.put("sourceNatVlanNetmask", sourceNatVan.getVlanNetmask());
             sourceNatDetails.add(sourceNatDetail);
         }
-        try {
-            try {
-                HostVO nuageVspHost = getNuageVspHost(config.getPhysicalNetworkId());
-                ApplyStaticNatVspCommand cmd = new ApplyStaticNatVspCommand(networkDomain.getUuid(), vpcOrSubnetUuid, isL3Network, sourceNatDetails);
-                ApplyStaticNatVspAnswer answer = (ApplyStaticNatVspAnswer)_agentMgr.easySend(nuageVspHost.getId(), cmd);
-                if (answer == null || !answer.getResult()) {
-                    s_logger.error("ApplyStaticNatNuageVspCommand for network " + config.getUuid() + " failed");
-                    if ((null != answer) && (null != answer.getDetails())) {
-                        throw new ResourceUnavailableException(answer.getDetails(), Network.class, config.getId());
-                    }
-                }
-            } catch (Exception e) {
-                s_logger.warn("Failed to apply static Nat in Vsp " + e.getMessage());
+
+        HostVO nuageVspHost = getNuageVspHost(config.getPhysicalNetworkId());
+        ApplyStaticNatVspCommand.Builder cmdBuilder = new ApplyStaticNatVspCommand.Builder().networkDomainUuid(networkDomain.getUuid())
```

Comment:
Consider placing each method call to the builder on a single line to make the code more readable.